Context/Context.php

```diff
@@ -94,19 +115,16 @@ public static function setValue(ContextKey $key, $value, $parent=null)
     /**
      * Fetch a value from the Context given a key value.
      *
-     * @param ContextKey $key
-     *
-     * @throws ContextValueNotFoundException
-     * @return mixed
-     * @suppress PhanUndeclaredClassMethod
+     * @return mixed|null
      */
     public function get(ContextKey $key)
     {
         if ($this->key === $key) {
             return $this->value;
         }
+
         if (null === $this->parent) {
-            throw new ContextValueNotFoundException();
+            return null;
```

Comment:
This relates to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/error-handling.md. Allows returning an invalid span in the case of there not being an active span. Checking for `null` seemed better than catching an exception.